### DISCUSSION TITLE
Is there a way to launch a new tab and automatically connect to a ssh server?

### DISCUSSION THREAD

**Discussion by tv21** [Eugeny/tabby#3216] [Q&A]
What I am looking for is a feature similar to iTerm2 on the Mac, where you can create a "profile" for each server you regularly connect to, and then when you click on that profile name it will launch a new tab and connect you to the server using the ssh command you specified in the profile.  This could also be thought of as a favorites or bookmarks feature, the point being that you can connect to a server by simply clicking on the server name and it will open it in a new tab (I realize some people may prefer a new tile or new window, but I like my sessions in tabs).  In iTerm2 you can select the profile you want from the "Profiles" dropdown menu in the top menu bar, or you can do what I do and have a sidebar that always shows a list of your profiles so it is literally one point-and-click to open one.  That is my preferred behavior and something I'd very much like to emulate, but a dropdown in the top menu would be my second choice.

In your online demo I see that you can create additional profiles but what I cannot figure out is how you launch them!  Is there any way to do what I am attempting in Terminus?

**Comment by Eugeny** [comment]
Sure, if you enable the built-in SSH client (disabled by default on Mac under Settings -> Plugins), you can then add your connections under Settings -> SSH and connect to them through the "SSH" button in the tab bar.

Alternatively, set your SSH commandline as a new profile's shell in Settings -> Shell
